.github/workflows/build.yml

```diff
@@ -30,4 +30,9 @@ jobs:
       - name: Build Leo
         id: build-leo
         run: |
-          npm run transform-tokens
\ No newline at end of file
+          npm run transform-tokens
+
+      - name: Format
+        id: format
+        run: |
+          npx prettier ./ --list-different
```

Comment:
Since we're specifying `prettier` as a dependency, can we run it via `node` / `npm` instead of `npx`?

Reply:
We can yeah. I've added a `format-diff` script which we can run during the build workflow. FWIW, I'm pretty sure `npx` will use the installed version of the package, and if it isn't installed, it will ask if you want to download it.